### PR TITLE
readme: add notice about maintenance stopping

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

File: candidtest/idmtest.go [hunk 1]
```diff
@@ -24,8 +24,8 @@ import (
 	"gopkg.in/macaroon-bakery.v2/httpbakery/agent"
 	macaroon "gopkg.in/macaroon.v2"
 
-	"gopkg.in/CanonicalLtd/candidclient.v1"
```
Comment (frankban) [inline candidtest/idmtest.go]: Why this change?

**Review by frankban — APPROVED** [review]

**Review by fabricematrat — APPROVED** [review]

**Review by alesstimec — APPROVED** [review]
